lib/iris/fileformats/netcdf.py

```diff
@@ -534,14 +534,13 @@ def _load_cube(engine, cf, cf_var, filename):
     names = {coord.var_name: coord.standard_name or coord.var_name or 'unknown'
              for coord in cube.coords()}
 
-    cube.cell_methods = [iris.coords.CellMethod(method=method.method,
-                                                intervals=method.intervals,
-                                                comments=method.comments,
-                                                coords=
-                                                [names[coord_name] if
-                                                 coord_name in names else
-                                                 coord_name for coord_name in
-                                                 method.coord_names])
+    cube.cell_methods = [iris.coords.CellMethod(
+                             method=method.method,
+                             intervals=method.intervals,
+                             comments=method.comments,
+                             coords=[names[coord_name]
+                                     if coord_name in names else coord_name
+                                     for coord_name in method.coord_names])
                          for method in cube.cell_methods]
```

Comment:
Mixed indentation methods. You've tried to do a step indent on top of a visual indent, and pep8 doesn't like it. You need to either do a visual indent to level with Method( - I assume this would take you to past 79chars, or - switch the whole statement to 4char step indenting.